doc/cascadia/profiles.schema.json

```diff
@@ -868,16 +929,38 @@
         },
         "icon": { "$ref": "#/definitions/Icon" },
         "name": {
-          "description": "The name that will appear in the command palette. If one isn't provided, the terminal will attempt to automatically generate a name.",
+          "description": "The name that will appear in the command palette. If one isn't provided, the terminal will attempt to automatically generate a name.\nif name is a string, it  will be the name of the command.\nif name is a object, the key property of the object will be used to lookup a localized string resource for the command",
```

Comment:
```suggestion
          "description": "The name that will appear in the command palette. If one isn't provided, the terminal will attempt to automatically generate a name.\nIf name is a string, it will be the name of the command.\nIf name is a object, the key property of the object will be used to lookup a localized string resource for the command",
```